### ISSUE TITLE
Marker with tooltip no longer visible with v0.8.2

### ISSUE THREAD

**Issue by rpastro** [vue-leaflet/vue-leaflet#269]
With v0.8.2, markers with a tooltip and without a custom icon are no longer rendered in the map.  This worked properly with v0.8.1.

```
    <l-marker
      v-for="(i, index) in items"
      :key="index"
      :lat-lng="[i.lat, i.lon]"
      @click="itemClicked(i)"
    >
      <l-tooltip>{{ i.tooltip }}</l-tooltip>
    </l-marker>
```

This seems to be a regression introduced by #173

**Comment by mikeu** [comment]
Hi @rpastro , thank you for reporting this! The same problem was also occurring when the only content of a marker was the `<l-popup>` component. I've just released [v0.8.3](https://github.com/vue-leaflet/vue-leaflet/releases/tag/v0.8.3) in which this problem is resolved. Please let me know if you still encounter any issues with it.

**Comment by kaboume** [comment]
I had the same issue with the 0.8.2 release.
I confirmed that it is fixed with the 0.8.3 release.
Thanks!

**Comment by stefanw** [comment]
I used the `icon` prop on `<LMarker :icon=...>` and the icon would not show if `LMarker` had no children (it worked before v0.8.2). I resolved it by using `LIcon` as a child of `LMarker` instead.
